spec.bs

```diff
@@ -32,6 +32,7 @@ spec: ECMASCRIPT; urlPrefix: https://tc39.es/ecma262/
     text: abrupt completion; url: sec-completion-record-specification-type
     text: throw completion; url: sec-completion-record-specification-type
     text: ScriptEvaluation; url: sec-runtime-semantics-scriptevaluation
+    text: Number; url: sec-ecmascript-language-types-number-type

```

Comment:
I think you *should* be able to use `{{Number}}` instead, like https://github.com/w3c/IndexedDB/blob/main/index.bs#L615. At least it [appears to be exposed](https://dontcallmedom.github.io/webdex/n.html#Number%40%40%40%40interface) here. Could you try that?

Reply:
That works... but it links towards Number /objects/ (https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-number-objects), not number primitive type (though my choice is probably wrong, too, it should probably be #sec-terms-and-definitions-number-value, not #sec-ecmascript-language-types-number-type).  Which is probably not what IndexedDB wants, either.

Reply:
Hmm OK, then I think it is OK to leave this as-is.